bucket/mpc.json

```
@@ -0,0 +1,21 @@
+{
+    "version": "0.22",
+    "description": "Command-line music player client for mpd",
+    "homepage": "https://www.musicpd.org/clients/mpc", 
```

```suggestion
    "homepage": "https://www.musicpd.org/clients/mpc",
```
There is a trailing whitespace, which causes CI to fail.